tabpy/tabpy_server/handlers/base_handler.py

```diff
@@ -434,11 +434,11 @@ def fail_with_auth_error(self):
                 log_message="Invalid credentials provided.",
             )
         else:
-            self.logger.log(logging.ERROR, "Failing with 400 for Bad Request")
-            self.set_status(400)
+            self.logger.log(logging.ERROR, "Failing with 406 for not acceptable")
```

Comment:
```suggestion
            self.logger.log(logging.ERROR, "Failing with 406 for Not Acceptable")
```
nit: I think the names of the HTTP error codes are usually written as all caps

Reply:
Hi there! Thanks for the comment. I was thinking of matching the existing "unauthorized request" format, preceding  on line 428, but I like your suggestion. Let's go with that.